scripts/fix-agents-frontmatter.sh

```
@@ -0,0 +1,155 @@
+#!/bin/bash
+# Script to add YAML frontmatter to agents that are missing it
+
+AGENTS_DIR="/Users/colinaulds/Desktop/projects/flrts/.claude/agents"
+
+echo "Fixing agent frontmatter in $AGENTS_DIR"
+
+# Function to add frontmatter to an agent file
+add_frontmatter() {
+    local file="$1"
+    local basename=$(basename "$file" .md)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)